Source/ACE.Server/Managers/LandblockManager.cs

```diff
@@ -212,5 +220,28 @@ public static void UnloadLandblocks()
                 }
             }
         }
+
+        public static void Initialize()
+        {
+            var landBlockIdList = new List<LandblockId>();
+
+            for (uint x = 0; x < RawLandblockId.Length; x++)
+            {
+                var landBlockId = new LandblockId(RawLandblockId[x]);
+                landBlockIdList.Add(landBlockId);
+            }
+
+            foreach (var landBlockId in landBlockIdList)
+            {
+                Stopwatch sw = Stopwatch.StartNew();
```

Comment:
You don't really need the stopwatch stuff here unless you're really interested in logging the time.